amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -571,6 +603,12 @@ function viewDifferences(activityOneId) {
 </script>
 </digi:form>
 <digi:form action="/compareActivityVersions.do" method="post" type="aimCompareActivityVersionsForm">
-	<input type="hidden" name="activityOneId" id="activityOneId" />
-	<input type="hidden" name="method" id="method" />
-</digi:form>
+	<input type="hidden" name="activityOneId" id="activityOneId" /> 	 
+	<input type="hidden" name="method" id="method" />	
+	

```

Comment:
Please do not add extra lines that are not needed, makes the code review difficult